src/main/scala/rocks/muki/graphql/codegen/ScalametaUtils.scala

```diff
@@ -0,0 +1,13 @@
+package rocks.muki.graphql.codegen
+
+import scala.meta._
+
+object ScalametaUtils {
```

Comment:
Not sure if we should wait to upgrade scalameta until there's a proper way to replace this.
This looks very hack :stuck_out_tongue:

Reply:
I agree it looks quite hacky. Source https://gitter.im/scalameta/scalameta?at=5b9ba14f8909f71f75d1b4bd  @olafurpg

Reply:
You can use `parse[Term] match { case Parsed.Success(ref: Term.Ref) =>` instead if that's nicer, or construct it by hand with `split('.').foldLeft()` as it's currently in this PR. We could have a `parse[Term.Ref]` helper but you would still need to match against `Parsed.Success`.

Parsing strings into a data structure like `Term.Ref` is inherently unsafe since the string can contain anything! For example, we don't want code like below 😜 

```scala
package println("hello world!")

import 1 + 2
```

Reply:
I end up constructing `Term.Ref` by hand. https://github.com/muuki88/sbt-graphql/pull/53/files#diff-ab59f451a41d0da55b80a446bd82614cR28 It works in this particular case.